internal/queries/cart.go

```diff
@@ -87,6 +87,72 @@ func (q *CartQueries) Carts() ([]*models.Cart, error) {
 	return carts, nil
 }
 
+// Carts is ...
+func (q *CartQueries) Cart(cartId string) (*models.Cart, error) {
+
+	query := `
+	SELECT 
+    id, 
+    email, 
+    name, 
+    amount_total,
+    currency,
+    payment_id,
+    payment_status,
+    strftime('%s', created),
+    strftime('%s', updated)
+	FROM cart
+	WHERE id = ?
+	`
+
+	rows, err := q.DB.QueryContext(context.TODO(), query, cartId)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	var email, name, paymentID sql.NullString
+	var updated sql.NullInt64
+	cart := &models.Cart{}
+
+	err = rows.Scan(
+		&cart.ID,
+		&email,
+		&name,
+		&cart.AmountTotal,
+		&cart.Currency,
+		&paymentID,
+		&cart.PaymentStatus,
+		&cart.Created,
+		&updated,
+	)
+	if err != nil {
+		return nil, err
+	}
+
+	if email.Valid {
+		cart.Email = email.String
+	}
+
+	if name.Valid {
+		cart.Name = name.String
+	}
+
+	if paymentID.Valid {
+		cart.PaymentID = paymentID.String
+	}
+
+	if updated.Valid {
+		cart.Updated = updated.Int64
+	}
```

Comment:
I think that validating data from the database here is unnecessary.
By logic - when we write data to the database, we validate them, and they should return already valid.